### DISCUSSION TITLE
scatter gather functionality for ndarray

### DISCUSSION THREAD

**Discussion by vansweej** [rust-ndarray/ndarray#871] [General]
Hi,

Maybe I am overseeing things in the documentation but is there a way to do scatter/gather operations without using iteration functionality (like map). I mean to have it as a direct api call.
I do it now with a map and it works well, but I am wondering if I am overseeing something in the documentation.
Off course nothing prevents me from hiding the implementation with map behind a simple function interface but I am asking first.

Jan

**Comment by bluss** [comment]
Hi, in what sense do you mean scatter/gather here, do you have something more concrete to use for explaining? For example that wrapper method?

**Reply by vansweej** [reply]
Hi,

I am very sorry for the late reply. A burnout related to lock down kept me from keeping up with my socials.

I found in the ndarray api what I needed. What I tried describe above was the functionality offered by `select` and I am using that now.
More questions to come.

Jan
